### DISCUSSION TITLE
Cannot start taskcafe on QNAP Container station

### DISCUSSION THREAD

**Discussion by neon1000** [JordanKnott/taskcafe#153] [Q&A]
Hi all, 

I cannot start the app from docker, the following error comes up:

ERRO[07-04-2022 13:10:07] issue connecting to database, retrying        error="pq: unknown response for startup: '\\\\'" retryDuration=0s retryNumber=0
ERRO[07-04-2022 13:10:07] issue connecting to database, retrying        error="pq: unknown response for startup: '\\\\'" retryDuration=2s retryNumber=1
ERRO[07-04-2022 13:10:09] issue connecting to database, retrying        error="pq: unknown response for startup: '\\\\'" retryDuration=4s retryNumber=2
ERRO[07-04-2022 13:10:13] issue connecting to database, retrying        error="pq: unknown response for startup: '\\\\'" retryDuration=6s retryNumber=3
Error: pq: unknown response for startup: '\\'

I hope anyone can help me and sorry for the maybe stupid question but i am a complete beginner with the use of docker files :-(

Thanks for anyone who can give me a hint!

**Comment by GwynethLlewelyn** [comment]
From what I read on other threads, it's probable that Taskcafé just runs under Docker for Intel CPUs; it's likely that QNAP (as so many other embedded devices) uses an ARM processor. You can change some parameters on Docker to get a different entry point, i.e. replacing the environment so that it supports the ARM architecture instead. There is a port of Taskcafé that aims to do that, but unfortunately it's currently not being actively worked on.

You can always try to compile & run Taskcafé outside Docker :-) (good luck with that, it's way tougher than I thought it would be...)
